src/tasks/installers/helm.ts

```diff
@@ -333,6 +334,13 @@ error: E_COMMAND_FAILED`)
       setOptions.push(`--set global.chePostgresPVCStorageClassName=${flags['postgres-pvc-storage-class-name']}`)
     }
 
+    if (flags.multiuser) {
+      // Generate Keycloak admin password
+      const keycloakPassword = generatePassword(12)
+      setOptions.push(`--set che-keycloak.keycloakAdminUserPassword=${keycloakPassword}`)
+      ctx.highlightedMessages.push(`Autogenerated Keycloak credentials is: "admin:${keycloakPassword}"`)
```

Comment:
is -> are

Reply:
`The autogenerated`

Reply:
Could you provide the full `chectl` output?

Reply:
```
  ✔ Show important messages
    ✔ ❗[MANUAL ACTION REQUIRED] Please add Che self-signed CA certificate into your browser: /home/mmorhun/cheCA.crt.
   Documentaton how to add a CA certificate into a browser: https://www.eclipse.org/che/docs/che-7/installing-che-in-tls-mode-with-self-signed-certificates/#using-che-with-tls_installing-che
-in-tls-mode-with-self-signed-certificates
    ✔ Autogenerated Keycloak credentials are: "admin:42uE9uFZdGZZ"
Command server:start has completed successfully.
```